app/scripts/directives/configEditor.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+
+angular.module('openshiftConsole').component('configEditor', {
+    bindings: {
+      mobileClient: '<',
+    },
+    templateUrl: 'views/config-editor.html',
+    controller: [
+      'APIService', 
+      'DataService', 
+      'API_CFG', 
+      ConfigEditorCtrl]
+  });
+
+function ConfigEditorCtrl(APIService, DataService, API_CFG) {
+  var ctrl = this;
+  var watches = [];
+  ctrl.$onInit = function(){
+    var context = {namespace: _.get(ctrl, "mobileClient.metadata.namespace")};
+    //keep list of active $scope.services upto date
+    DataService.watch(
+      APIService.getPreferredVersion('serviceinstances'), 
+      context, 
+      function (serviceinstances){
+        ctrl.services = _.filter(serviceinstances.by("metadata.name"), function(service, serviceName){
+          return _.indexOf(ctrl.mobileClient.spec.excludedServices, serviceName) === -1;
+        });
+        DataService.list(APIService.getPreferredVersion('configmaps'), context, updateConfig, {errorNotification: false});
+      }, 
+      { errorNotification: false }
+    );
+    watches.push(DataService.watch(APIService.getPreferredVersion('configmaps'), context, updateConfig, {errorNotification: false}));
+
+    // update the config string by pulling out configmaps that match $scope.services
+    function updateConfig(configmaps){
+      var configs = [];
+      _.each(configmaps.by("metadata.name"), function(configmap, configmapName){
+        if(_.findIndex(ctrl.services, {metadata: {labels: {serviceName: configmapName}}}) !== -1){
+          if(_.get(configmap, "metadata.labels.mobile") === "enabled"){
+            configs.push({
+              id: configmapName,
+              name: configmapName,
+              type: configmap.data.type,
+              url: configmap.data.uri,
+              config: configmap.data
+            });
+          }
+        }
+      });
+      var ret = {
+        "version": 1,
```

Comment:
Is this always `1`?

Reply:
it is hardcoded as 1 in the mobile-cli, where I copied it from